pkg/xray/resource_xray_license_policy.go

```diff
@@ -20,143 +21,173 @@ func resourceXrayLicensePolicyV2() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "Name of the policy (must be unique)",
 			},
 			"description": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "More verbose description of the policy",
 			},
 			"type": {
-				Type:     schema.TypeString,
-				Required: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Type of the policy",
 			},
 			"author": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "User, who created the policy",
 			},
 			"created": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Creation timestamp",
 			},
 			"modified": {
-				Type:     schema.TypeString,
-				Computed: true,
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Modification timestamp",
 			},
 			"rules": {
-				Type:     schema.TypeList,
-				Required: true,
+				Type:        schema.TypeList,
+				Required:    true,
+				Description: "Nested block describing the policy rules. Described below",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"name": {
-							Type:     schema.TypeString,
-							Required: true,
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Name of the rule",
 						},
 						"priority": {
-							Type:     schema.TypeInt,
-							Required: true,
+							Type:        schema.TypeInt,
+							Required:    true,
+							Description: "Integer describing the rule priority",
```

Comment:
Do we need validation for this field? What are the acceptable values?

Reply:
Policy can have multiple rules, when you add them in the UI, `priority` it's the order number for each of them. 
You can reassign the priority. Documentation doesn't say anything about the maximum number of rules in one policy, so it's not clear what to verify, except that the value is an integer

Reply:
I've added `validation.ToDiagFunc(validation.IntAtLeast(1)),`